### DISCUSSION TITLE
现在是不是已经无法区别房管的弹幕了？。

### DISCUSSION THREAD

**Discussion by HuxiaoRoar** [xfgryujk/blivechat#218] [提问] [answered]
哎，怪我当时没有仔细看软件介绍，折腾完了，才发现房管现在已经没有那个额外的属性，没办法通过CSS定位，然后我看软件的主页写到。高亮舰队、 房管(删除线) 、主播的用户名。这是B站那方的改动吗？

**Comment by xfgryujk** [comment, marked as answer]
用房间ID连接才能区分房管

**Reply by HuxiaoRoar** [reply]
可惜，这样就没法看见昵称和头像、小小的遗憾吧。
